README.md

```diff
@@ -25,9 +26,9 @@ Recommandation](http://www.ivoa.net/documents/latest/UCDlistMaintenance.html).
 ## Status?
 
 The last stable version is
-**[REC-1.3](http://www.ivoa.net/documents/UCD1+/20180527/index.html)**.
+**[EN-1.4](https://ivoa.net/documents/UCD1+/20210430/index.htmll)**.
 
-Next version will be 1.4, as an _Endorsed Note_. It is prepared in this reporsitory. 
+Next version will be 1.5, as an _Endorsed Note_. It is prepared in this reporsitory. 
```

Comment:
Followup of my previous comment, 1.5 should be here listed as Proposed Endorsed Note consequently

Reply:
That is a good question. If we have an _Endorsed Note_, and we update it to a new minor version, it has to be proposed as a _Proposed Endorsed Note_ to TCG, and, if accepted, be switched to _Endorsed Note_ at this time. Is this the process?

Reply:
That is (at least) my understanding.
It's like having a WD for version N.M+1 after you reach REC-N.M.